integrations/docker/images/chip-build-imx/Dockerfile

```diff
@@ -22,6 +22,7 @@ RUN set -x \
 
 FROM connectedhomeip/chip-build:${VERSION}
 
+
 COPY --from=build /opt/fsl-imx-xwayland /opt/fsl-imx-xwayland
 
-ENV IMX_SDK_ROOT=/opt/fsl-imx-xwayland/5.15-kirkstone/
+ENV IMX_SDK_ROOT=/opt/fsl-imx-xwayland/6.1-langdale/
```

Comment:
you need to change `integrations/docker/images/chip-build-vscode/Dockerfile` as well (both the copy and env)

Reply:
@andy31415 I think I got it?

Reply:
> you need to change `integrations/docker/images/chip-build-vscode/Dockerfile` as well (both the copy and env)

Thank you for the comments, it is fixed in the updated PR.